webanno/webanno
# #1394 - Telemetry support

**What's in the PR**
- Instance ID generation
- Pluggable telemetry support
- Modularization of page footer

**How to test manually**
* Start the application, log in as admin and see if telemetry config comes up
* Configure telemetry via the heart icon in the footer

**Automatic testing**

**Documentation**
